openedx/core/djangoapps/schedules/admin.py

```diff
@@ -11,11 +11,46 @@ class ScheduleExperienceAdminInline(admin.StackedInline):
 
 @admin.register(models.Schedule)
 class ScheduleAdmin(admin.ModelAdmin):
-    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline')
+    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline', 'experience_display')
+    list_filter = ('experience__experience_type', 'active')
     raw_id_fields = ('enrollment',)
     readonly_fields = ('modified',)
-    search_fields = ('enrollment__user__username', 'enrollment__course_id',)
+    search_fields = ('enrollment__user__username', 'enrollment__course__id',)
     inlines = (ScheduleExperienceAdminInline,)
+    actions = ['deactivate_schedules', 'activate_schedules', 'set_experience_to_default', 'set_experience_to_course_updates']
+
+
+    def deactivate_schedules(self, request, queryset):
+        rows_updated = queryset.update(active=False)
```

Comment:
Are we good about checking the active flag? We might not even be checking it in the resolver...

Reply:
We check if the enrollment is active, but not the schedule, it looks like.

Do we need the active flag? What does it mean if a schedule is deactivated? It should just be treated as if it doesn't exist?

Reply:
Ok, I added a check for active in the resolvers.

Reply:
We probably need to check "active" here too:

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/models.py#L1763